scripts/build.sh

```diff
@@ -19,8 +19,14 @@ while test $# -gt 0; do
 done
 
 [ -d vendor/github.com/ligato/crd-example/ ] && (echo "Run: rm -rf vendor/github.com/ligato/crd-example;dep ensure";exit 1)
-test -z "$(go fmt ./...)" || (echo "Run go fmt ./... and recommit your code";exit 1)
-go get -u github.com/golang/protobuf/protoc-gen-go
-go generate ./...
-go install ./...
-go test $RACE_ENABLED ./...
+test -z "$(env GO111MODULE=on go fmt ./...)" || (echo "Run go fmt ./... and recommit your code";exit 1)
+env GO111MODULE=on go get -u github.com/golang/protobuf/protoc-gen-go
```

Comment:
Perhaps, you could just add `export GO111MODULE=on` to the beginning of the file.